internal/trace/trace.go

```diff
@@ -53,6 +113,18 @@ func toStatus(err error) trace.Status {
 	}
 }
 
+// toOpenTelemetryStatus converts an error to an equivalent OpenTelemetry status description.
+func toOpenTelemetryStatusDescription(err error) string {
+	var err2 *googleapi.Error
```

Comment:
Should this just be an APIError now? Everything should return those now, right? Same with the above toStatus stuff

Reply:
I'll look into this, thanks!

Reply:
I think this logic in bigtable shows a situation where the `err` closing the span could be an unwrapped `Status` or even a `io.EOF`:

https://github.com/googleapis/google-cloud-go/blob/bigtable/v1.20.0/bigtable/bigtable.go#L182-L215

Reply:
sure, but I just meant L117 here. Other cases can still be left.

Reply:
ah sorry, i misunderstood

Reply:
After trying this, I think it's best to just extract the `googleapi.Error` (if found), since it exposes the HTTP error message. With APIError, that message is only exposed by `Error()` but it gets some added prefix:

```
 trace_test.go:131: got googleapi: Error 400: INVALID ARGUMENT, want INVALID ARGUMENT
```

Reply:
I think that would be fine to call Error and all HTTP APIError *should have wrap a googleapis.Error. But I am fine with this for now. We can always make that change later.